File: cv.php
<!-- https://github.com/mustangostang/spyc/ -->

<?php

require_once "Spyc.php";


class Cv
{

    public function setup()
    {
        $this->yamlParser = new YamlParser();
    }

    public function createYAML()
    {
        $yaml = Spyc::YAMLDump(array(
            "name" => "VALLEE",
            "firstname" => "Hugo",
            "age" => 23,
            "skill" => "M2 MIAGE",
            "employed" => False,
            "favouriteWebsite" => "OpenCompare",
            "languages" =>
                array(
                    "java" => "Pro",
                    "c#" => "Not so bad",
                    "YAML" => "Very bad"
                )
        ), 4, 60);
    }
}
?>

File: YAML-CV/cv.java
/* Using https://github.com/EsotericSoftware/yamlbeans */

class MyCV {

    public static void main (String[] args){

        CV cv = new CV();
        cv.name = "VALLEE";
        cv.firstname = "Hugo";
        cv.age = 23;
        cv.skill = "M2 MIAGE";
        cv.favouriteWebsite = "OpenCompare";
        language[] languages = {"java" => "pro", "c#" => "not so bad", "YAML" => "Very bad"};
        cv.languages = languages;
        YamlWriter writer = new YamlWriter(new FileWriter("cv.yml"));
        writer.write(cv);
        writer.close();
    }
}